assets/wizards/performance/views/push-notifications/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Performance Wizard Push Notifications screen.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { ToggleControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { TextControl, withWizardScreen } from '../../../../components/src';
+
+/**
+ * Description and controls for the Push Notification feature.
+ */
+class PushNotifications extends Component {
+	/**
+	 * Render.
+	 */
+	render() {
+		const { settings, updateSetting } = this.props;
+		return (
+			<Fragment>
+				<p>
+					{ __(
+						'You just published new content and you want to let everyone know? Why not send a push notification? We has an integrated connection to Firebase that lets you manage registered devices and send push notifications to all or selected devices!'
```

Comment:
```suggestion
						'You just published new content and you want to let everyone know? Why not send a push notification? We have an integrated connection to Firebase that lets you manage registered devices and send push notifications to all or selected devices!'
```

Reply:
Fixed in https://github.com/Automattic/newspack-plugin/pull/141/commits/9b0344904db4b1d4fe2ed31030e628d296d1349c